### DISCUSSION TITLE
VoiceAttack after shutting down stays in memory when using EDDI plugin

### DISCUSSION THREAD

**Discussion by stevan54** [EDCD/EDDI#2635] [EDDI VoiceAttack Q&A] [answered]
Is there any workaround for this?

**Comment by Tkael** [comment]
I believe I was able to make some progress on this on the last release. Which version are you using?

**Reply by stevan54** [reply]
I'm using VoiceAttack v1.14
Eddi v4.04

**Comment by stevan54** [comment, marked as answer]
I downloaded EDDY 4.1.0-b2 and now Voiceattack shuts down properly and removes itself from memory.
